lxd/events/events.go

```diff
@@ -60,13 +60,8 @@ func (s *Server) AddListener(group string, connection *websocket.Conn, messageTy
 }
 
 // SendLifecycle broadcasts a lifecycle event.
-func (s *Server) SendLifecycle(group, action, source string, context map[string]interface{}, requestor *api.EventLifecycleRequestor) error {
-	s.Send(group, "lifecycle", api.EventLifecycle{
-		Action:    action,
-		Source:    source,
-		Context:   context,
-		Requestor: requestor})
-	return nil
+func (s *Server) SendLifecycle(group string, event api.EventLifecycle) {
```

Comment:
Technically we could drop this function entirely and pass `state.State` into `Lifecycle` and just call `s.Events.Send()` directly.

Reply:
Indeed but I think I like the idea of having `SendLifecycle` which guarantees that you're sending it with the right event type and also that what it's passed is indeed an `EventLifecycle`.